up-transport-zenoh-cpp/release/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git
+from conan.tools.files import copy, get
+import os
+import yaml
+
+class upZenohTransportRecipe(ConanFile):
+    name = "up-transport-zenoh-cpp"
+
+    # Optional metadata
+    license = "Apache-2.0"
+    author = "Contributors to the Eclipse Foundation <uprotocol-dev@eclipse.org>"
+    url = "https://github.com/eclipse-uprotocol/up-transport-zenoh-cpp"
+    description = "This library provides a Zenoh-based uProtocol transport for C++ uEntities"
+    topics = ("automotive", "iot", "uprotocol", "messaging")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    test_requires = "gtest/1.14.0"
```

Comment:
Minor: this could also be specified in the conandata.yml and pulled in using the `requirements()` method